modules/policy-criteria.adoc

```diff
@@ -343,12 +343,13 @@ LOCALHOST
 
 
 | Drop Capabilities
-| Linux capabilities that must be dropped from the container.
-For example `CAP_SETUID` or `CAP_NET_RAW`.
+| Linux capabilities that must be dropped from the container. Provides alerts when the specified capabilities are not dropped.
+For example, if configured with `SYS_ADMIN` AND `SYS_BOOT`, and the deployment manifest YAML file includes only *one* or *neither* of these two capabilities, the alert occurs.
```

Comment:
s/includes/drops

The deployment yaml includes only one or neither in the drop capabilities section of the manifest == The deployment drops only one or neither of these two capabilities

Reply:
@kcarmichael08 What do you think about the suggestion above ^? Maybe it got missed in the shuffle.